ts/WoltLabSuite/Core/Ui/Feed/Dialog.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * Shows a dialog with an anonymous and personalized link for RSS feeds with access tokens.
+ *
+ * @author  Matthias Schmidt
+ * @copyright  2001-2021 WoltLab GmbH
+ * @license  GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @module  WoltLabSuite/Core/Ui/Feed/Dialog
+ */
+
+import UiDialog from "../Dialog";
+import * as StringUtil from "../../StringUtil";
+import * as Language from "../../Language";
+import * as Clipboard from "../../Clipboard";
+import * as UiNotification from "../Notification";
+
+/**
+ * Copies one of links to the clipboard.
+ */
+async function copy(event: Event): Promise<void> {
+  event.preventDefault();
+
+  const target = event.currentTarget as HTMLElement;
+  const input = target.parentNode!.querySelector('input[type="text"]') as HTMLInputElement;
+
+  await Clipboard.copyTextToClipboard(input.value);
+
+  UiNotification.show(Language.get("wcf.global.rss.copy.success"));
+}
+
+/**
+ * Opens the dialog with an anonymous and personalized link after clicking on the RSS feed link.
+ */
+function openDialog(event: Event): void {
+  event.preventDefault();
+
+  const alternative = event.currentTarget as HTMLAnchorElement;
+  const linkWithAccessToken = alternative.href;
+
+  const linkWithoutAccessToken = linkWithAccessToken.replace(/at=[^&]*&/, "&").replace(/(\\?|&)at=[^&]*/, "");
```

Comment:
I believe this will break links that look like the following:

```
https://example.com/forum/rss-feed/?at=123&foo=bar
```
Because the `?` will be stripped. The second replacement probably should replace by `$1` and a third replacement should be added that removes `(\?|&)$`.